repos/rust-lang/thorin.toml

```diff
@@ -0,0 +1,15 @@
+org = 'rust-lang'
+name = 'thorin'
+description = 'DWARF packaging utility, written in Rust, supporting GNU extension and DWARF 5 package formats.'
+bots = ["rustbot"]
+
+[access.teams]
+compiler = 'maintain'
+compiler-contributors = 'maintain'
+
+[[branch-protections]]
+pattern = "main"
+ci-checks = ['continuous integration']
```

Comment:
I *think* that this should be `test`, based on the name of the job, not on the workflow (https://github.com/rust-lang/thorin/blob/main/.github/workflows/ci.yml#L7C15-L7C15).